Src/IronPython.SQLite/Connection.cs

```diff
@@ -367,8 +367,8 @@ private static void setResult(Sqlite3.sqlite3_context ctx, object result)
                     Sqlite3.sqlite3_result_int64(ctx, (int)result);
```

Comment:
Maybe change to a switch statement?

Reply:
I thought about it but decided against since the goal of the PR is to merge in the change and not code cleanup.